tests/CommunityToolkit.Aspire.Hosting.SurrealDb.Tests/SurrealDbPublicApiTests.cs

```diff
@@ -0,0 +1,247 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting;
+
+namespace CommunityToolkit.Aspire.Hosting.SurrealDb.Tests;
+
+public class SurrealDbPublicApiTests
```

Comment:
Let's get some tests in here that assert all the right annotations are setup across the container, the mounts, args, etc.

Reply:
Do you have an example? I suppose Verify would be a useful for this kind of tests?